spek-runner/junit5/src/main/kotlin/org/spekframework/spek2/junit/SpekTestEngine.kt

```diff
@@ -49,6 +49,17 @@ class SpekTestEngine : TestEngine {
             .map { it.toString() }
 
         val classSelectors = discoveryRequest.getSelectorsByType(ClassSelector::class.java)
+            .filter {
+                // get all super classes
+                val superClasses = mutableListOf<String>()
+                var current = it.javaClass.superclass
+                while (current != null) {
+                    superClasses.add(current.name)
+                    current = current.superclass
+                }
+
+                superClasses.contains("org.spekframework.spek2.Spek")
```

Comment:
Can I suggest you use [`Spek::class.qualifiedName`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.reflect/-k-class/qualified-name.html) here? This will mean the name will update automatically if it is ever moved to another package.

Reply:
I've tried it before and it didn't work when running tests via Gradle. It was complaining about the class `Spek` not being in the classpath. I haven't investigated it more, but `Spek` should be in the classpath since the test engine depends on `spek-runtime` - not entirely sure why the error is happening.